draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -225,39 +240,46 @@ Key Binding is an optional feature. When Key Binding is required by the use-case
 
 Note: How the public key is included in SD-JWT is out of scope of this document. It can be passed by value or by reference.
 
-For presenting an SD-JWT to a Verifier that enforces Key Binding, the Holder
-creates a signed document, the Key Binding JWT as defined in (#kb-jwt), using
-its private key. This document contains a nonce to ensure the freshness of the signature, an audience value to
-indicate the intended audience for the document, and a hash that ensures the
-integrity of the data sent from the Holder to the Verifier. Details of the format of Key Binding JWTs are
-described in (#kb-jwt).
+When a Verifier requires Key Binding, the Holder presents an SD-JWT-KB covering the
+relevant SD-JWT.  The SD-JWT-KB encodes the SD-JWT as well as a Key Binding JWT.
+The Key Binding JWT encodes a signature by the Holder's private key over:
+
+* A hash of the SD-JWT
+* A nonce to ensure the freshness of the signature, and
+* An audience value to indicate the intended audience for the document
+
+Details of the format of Key Binding JWTs are described in (#kb-jwt).
 
 ## Verification
 
 At a high level, the Verifier
 
- * receives the SD-JWT from the Holder and verifies its signature using the Issuer's public key,
- * verifies the Key Binding JWT, if Key Binding is required by the Verifier's policy, using the public key included in the SD-JWT,
+ * receives either an SD-JWT or an SD-JWT-KB from the Holder
+ * verifies the signature on the SD-JWT (or the the SD-JWT inside the SD-JWT-KB) using the Issuer's public key,
+ * verifies the signature on the KB-JWT using the public key included (or referenced) in the SD-JWT, if an SD-JWT-KB was provided and the Verifier's policy requires Key Binding
```

Comment:
```suggestion
 * receives either an SD-JWT or an SD-JWT-KB from the Holder,
 * verifies the signature on the SD-JWT (or the the SD-JWT inside the SD-JWT-KB) using the Issuer's public key, and
 * verifies the signature on the KB-JWT using the public key included (or referenced) in the SD-JWT, if the Verifier's policy requires Key Binding.
```
Suggest to remove the "if an SD-JWT-KB was provided" part as it may be misleading.